railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -14,6 +14,7 @@ Rails.application.configure do
   # recommended that you enable it in continuous integration systems to ensure eager
   # loading is working properly before deploying your code.
   config.eager_load = ENV["CI"].present?
+  config.rake_eager_load = ENV["CI"].present?
```

Comment:
i did a similar thing in my test environment when i realized that. i didn't have the energy to explain all this in a rails PR. Sometimes i thank AI for this purpose.

I would suggest doing this instead though: 
```ruby
 config.eager_load = ENV["CI"].present?
 config.rake_eager_load = config.eager_load
```

So that we don't duplicate the same logic if a developer needs to change it. and it also clearly states that we want the same configuration as `config.eager_load` 👍🏼

Reply:
Ah yeah, not a bad idea! I'll leave to Rails maintainers, I think, unless you wanted to consider another PR?